app/templates/components/public/sponsor-list.hbs

```diff
@@ -1,9 +1,7 @@
 <h3 class="ui header">{{t 'Sponsors'}}</h3>
-{{#each-in sponsorsGrouped as |key sponsors|}}
```

Comment:
First you need to make the change mentioned above. and modify the computed property accordingly. Follow the discussion in #469 . A very similar thing was discussed there. Please follow discussions in _all_ PRs.